foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/SparkDockerPomGenerator.java

```diff
@@ -94,25 +94,16 @@ public void generate(GenerationContext context) {
 
             final String sparkOperatorAppName = "spark-operator";
             final String sparkInfrastructureAppName = "spark-infrastructure";
-            final String sparkWorkerAppName = "spark-worker-image";
 
             manualActionNotificationService.addNoticeToAddModuleToParentBuild(context, sparkDockerArtifactId, "docker");
             manualActionNotificationService.addDeployPomMessage(context, "aissemble-spark-operator-deploy-v2", sparkOperatorAppName);
             manualActionNotificationService.addDeployPomMessage(context, "aissemble-spark-infrastructure-deploy-v2", sparkInfrastructureAppName);
-            manualActionNotificationService.addDeployPomMessage(context, "aissemble-spark-worker-deploy", sparkWorkerAppName);
-            manualActionNotificationService.addSparkWorkerDockerBuildTiltMessage(context, sparkWorkerAppName, sparkDockerArtifactId
-                    ,context.getArtifactId());
+            manualActionNotificationService.addSparkApplicationTiltMessage(context, "pipeline-spark-application");
             String pipelineArtifactId;
             for (Pipeline pipeline: dataFlowPipelines) {
                 pipelineArtifactId = deriveArtifactIdFromCamelCase(pipeline.getName());
                 manualActionNotificationService.addSparkWorkerTiltResources(context, pipelinesArtifactId,
                         pipelineArtifactId, pipeline.getType().getImplementation());
-                if (pipeline.getType().getImplementation().equalsIgnoreCase("data-delivery-pyspark")) {
-                    manualActionNotificationService.addLocalResourceTiltFileMessage(context, context.getArtifactId(), sparkDockerArtifactId, pipelineArtifactId, pipelineArtifactId, false);
```

Comment:
Think we can delete this method as well from ` manualActionNotificationService`

Reply:
same as above, this is still used by inference/training pipeline. Will remove it then.